versions.rst

```diff
@@ -40,17 +39,36 @@ Unsupported versions
    :file: include/end-of-life.csv
 
 
+.. _versions-chart-all:
+
+Full chart
+==========
+
+.. raw:: html
+   :file: include/release-cycle-all.svg
+
+
 Status key
 ==========
 
-:feature: new features, bugfixes, and security fixes are accepted.
-:prerelease: feature fixes, bugfixes, and security fixes are accepted for the
-    upcoming feature release.
-:bugfix: bugfixes and security fixes are accepted, new binaries are still
-    released. (Also called **maintenance** mode or **stable** release)
-:security: only security fixes are accepted and no more binaries are released,
-    but new source-only versions can be released
-:end-of-life: release cycle is frozen; no further changes can be pushed to it.
+Python releases go through five phases:
+
+:feature: Before the first beta, the next full release can accept new features,
+   bug fixes, and security fixes.
+
+:prerelease: After the first beta, no new features can go in, but feature fixes
+   (including significant changes to new features), bug fixes, and security fixes
+   are accepted for the upcoming feature release.
+
+:bugfix: Once a version has been released, bug fixes and security fixes are
+   accepted. New binaries are built and released. (Also called **maintenance**
+   mode or **stable** release)
+
+:security: After two years, only security fixes are accepted and no more
```

Comment:
"two years" is only for 3.13+

Reply:
Change to "a few"

Reply:
To me `a few` seems both vague and higher than two.
Maybe just say `After 2 years (1.5 for versions <3.13), ...`?

Another option is keeping `After two years` and adding a footnote that explains that before 3.13 it was 1.5 and than it became 2, possibly linking the relevant PEPs/discussion.

Reply:
Updated, with some other tweaks.